src/platform/forms-system/src/js/utilities/review/index.js

```diff
@@ -45,15 +45,15 @@ export const openAndEditChapter = error => {
     `va-accordion-item[data-chapter="${error.chapterKey}"`,
   );
 
-  const accordionItemButton = accordionItem.shadowRoot.querySelector(
+  const accordionItemButton = accordionItem?.shadowRoot.querySelector(
     "button[aria-controls='content']",
   );
 
-  accordionItemButton.click();
+  accordionItemButton?.click();
 
   const pageTarget = findTargets(error).scroll;
 
-  const editButton = pageTarget.parentNode.querySelector('va-button');
+  const editButton = pageTarget?.parentNode.querySelector('va-button');
```

Comment:
The changes in this file and in the widgets & subtask fixes errors that show up when running tests with `--log-level=error`